models/stg_sage_intacct__ar_invoice_item.sql

```diff
@@ -1,3 +1,4 @@
+{{ config(enabled=var('sage_intacct__using_invoices', True)) }}
```

Comment:
Are deleted invoice items not tracked?

Reply:
I didn't see a fivetran_deleted column in the ar_invoice_item or ap_bill_item tables for some reason, so I didn't add the filter here